src/main/fc/config.c

```diff
@@ -483,6 +484,15 @@ void resetServoMixerConfig(servoMixerConfig_t *servoMixerConfig)
 }
 #endif
 
+#ifdef USE_MAX7456
+void resetMax7456Config(vcdProfile_t *pVcdProfile)
+{
+    pVcdProfile->video_system = 0; // AUTO see drivers/max7456.h
```

Comment:
Move video types enum `enum VIDEO_TYPES { AUTO = 0, PAL, NTSC };` into `vcd.h` and call define them as
`enum VIDEO_TYPES { VIDEO_TYPE_AUTO = 0, VIDEO_TYPE_PAL, VIDEO_TYPE_NTSC };`

Then set `pVcdProfile->video_system = VIDEO_TYPE_AUTO;` and remove comment.